docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -0,0 +1,28 @@
+= Customization
+
+This page will describe how you can customize your Keycloak instance.
```

Comment:
Same here, be more specific about what customizations we're talking about

Reply:
The specifics should be in the subtitles. For now there is only one customisation but we might have more in the near future.

Reply:
OK, let's see how that turns out